app/dashboard/models.py

```diff
@@ -4730,6 +4732,53 @@ def url(self):
     def get_absolute_url(self):
         return self.url()
 
+    def get_avg_rating(self, scale=1):
+        """Returns the average star ratings (overall, and total ratings) as well as a list of comments
+        for a particular user"""
+
+        feedbacks = HackathonProjectFeedbackEntry.objects.filter(receiver_project=self).all()
+
+        average_rating = {
+            'overall': sum([feedback.rating for feedback in feedbacks]) * scale / feedbacks.count() if feedbacks.count() != 0 else 0,
```

Comment:
since we already used `.all()` just use `len()` to count the length instead of `.count()`

Reply:
will address when we launch this feature, this has been removed.